tests/UI/campaigns/functional/BO/03_catalog/07_discounts/01_cartRules/01_CRUDCartRule/01_withAndWithoutCode.js

```diff
@@ -32,8 +32,20 @@ const {expect} = require('chai');
 let browserContext;
 let page;
 
+// Today date
+const today = new Date();
+// Day before
+const yesterday = (`0${today.getDate() - 1}`).slice(-2);
+// Current month
+const month = (`0${today.getMonth() + 1}`).slice(-2);
```

Comment:
What if the month is december ?

Reply:
Hi @sowbiba 
No problem for the getMonth() function because it always return a value between 0 and 11 so we need the "+1" to correctly format the date. But you make me think that the  " - 1" for the getDate() can cause some error because it return values between 1 and 31.
I will try another solution
Thanks @sowbiba :+1:

Reply:
It's correct :

March `02` with slice : `02`
December `011` with slice : `11`.